docs/examples/DefaultOptions.tsx

```diff
@@ -20,20 +16,10 @@ const promiseOptions = (inputValue: string) =>
     }, 1000);
   });
 
-export default class WithPromises extends Component<{}, State> {
-  state: State = { inputValue: '' };
```

Comment:
I've removed the state logic from this example because it wasn't necessary for the purpose of this example.